arangod/GeneralServer/RestHandler.cpp

```diff
@@ -221,6 +221,11 @@ futures::Future<Result> RestHandler::forwardRequest(bool& forwarded) {
 
     auto const& resultHeaders = response.response->messageHeader().meta();
     for (auto const& it : resultHeaders) {
+      if (it.first == "http/1.1") {
```

Comment:
I've no overview over the fuerte code. But shouldn't the status line be kept separately from the headers in the first place?

Reply:
And is the string `http/1.1` future proof enough?

Reply:
@goedderz : I don't know. But I think for now it is safe.

Reply:
its a hack to mimick some old behaviour of HttpClientConnection

Reply:
👍